lib/mods/clean-console.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const {getConsoleStatements} = require('./utils.js');
+
+/**
+ * Replaces console statements without ExpressionStatement parents
+ * with `undefined`.
+ *
+ * @param {any} j - the jscodemod api
+ * @param {Collection} root - the root Collection
+ *
+ * @return {Collection} - the modified root Collection
+ *
+ * @example
```

Comment:
This example does not seem useful, it's exactly the signature of the method. Similar below.